src/api/client.ts

```diff
@@ -81,6 +82,28 @@ export function getNostoClient(): PromiseLike<NostoClient> {
         }
     })
 }
+/**
+ * @group Nosto Client
+ * @category Core
+ */
+type LogLevel = 'error' | 'warn' | 'info' | 'debug'
+
+/**
+ * Logs an error to Nosto and the console
+ * @param msgOrError string message or error instance
+ * @param errorOrLevel error instance or log level string
+ * @param optLevel optional log level string
+ */
+export function log(msgOrError: string | unknown, errorOrLevel: LogLevel | unknown, optLevel?: LogLevel) {
```

Comment:
the overloaded function definitions are missing, this function is just the implementation